values.yaml

```diff
@@ -77,6 +77,29 @@ global:
   # Additionally requires Consul >= 1.4 and consul-k8s >= 0.10.1.
   bootstrapACLs: false
 
+  # consulNamespaces are an Enterprise only feature available in Consul v1.7+.
+  # A valid Consul Enterprise license is required to make use of this feature.
+  # Requires consul-k8s v0.x+
+  consulNamespaces:
+    enabled: false
+
+    # consulNamespaceName is the name of the Consul namespace to register all
+    # k8s services into. If the Consul namespace does not already exist,
```

Comment:
pending decision on splitting it up but if we don't split it then this should say "k8s services into for both Connect and Sync Catalog"